test/unit/client-side-encryption/autoEncrypter.test.js

```diff
@@ -3,42 +3,21 @@
 const fs = require('fs');
 const path = require('path');
 const sinon = require('sinon');
-const mongodb = require('mongodb');
-const BSON = mongodb.BSON;
-const EJSON = BSON.EJSON;
+const { MongoClient } = require('../../../src/mongo_client');
+const bson = require('bson');
+const { EJSON } = bson;
```

Comment:
```suggestion
const { MongoClient, BSON } = require('../../../src/mongo_client');
const { EJSON } = BSON;
```

Let's keep obtaining BSON from the driver here

Reply:
The mongo client doesn't export BSON.

Reply:
Whoops fixed, the driver export has it